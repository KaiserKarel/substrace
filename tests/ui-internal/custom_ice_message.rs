// rustc-env:RUST_BACKTRACE=0
// normalize-stderr-test: "Clippy version: .*" -> "Clippy version: foo"
// normalize-stderr-test: "internal_lints.rs:\d*:\d*" -> "internal_lints.rs"
// normalize-stderr-test: "', .*substrace_lints" -> "', substrace_lints"

#![deny(clippy::internal)]
#![allow(clippy::missing_clippy_version_attribute)]

fn it_looks_like_you_are_trying_to_kill_clippy() {}

fn main() {}
