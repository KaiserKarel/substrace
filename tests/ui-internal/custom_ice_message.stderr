thread 'rustc' panicked at 'Would you like some help with that?', substrace_lints/src/utils/internal_lints.rs
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

error: internal compiler error: unexpected panic

note: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust-clippy/issues/new

note: Clippy version: foo

query stack during panic:
end of query stack
